extra/1-currency-conversion.js

```diff
@@ -15,7 +19,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(PoundAmount) {
```

Comment:
recommend to use lower case for parameter

Reply:
I corrected.